deployer/templates/fluentd.yaml

```diff
@@ -63,6 +63,7 @@ objects:
             resources:
               limits:
                 cpu: 100m
+                memory: 512Mi
```

Comment:
Well I am sitting and waiting for it to hit it, but I think this will behave like any other part of OCP/Origin in that it will not have any more access to memory and the app may/may not crash (tends to happen with Java pods) and then get restarted by the RC/DC, or chug along but slowly (tends to happen with PHP pods). 

I just made the change in production and re-deployed one fluentD pod and am monitoring now, but looking at average usages, by eye not actually crunching numbers, most fluentd pods are consuming about 310MB, which seems reasonable enough, hence my 512Mi designation to give it room to grow.